test/integration/platform_admin_can_only_delete_cars_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class PlatformAdminCanOnlyDeleteCarsTest < ActionDispatch::IntegrationTest
+  test 'platform admin can only delete cars' do
+    create_cars(1)
+    login_platform_admin
+
+    visit cars_path
+
+    assert page.has_content?('Dave\'s Cars')
+    assert page.has_content?(100)
+    assert page.has_content?('El Camino')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.